mmv1/third_party/terraform/website/docs/guides/using_write_only_arguements.html.markdown

```diff
@@ -1,29 +1,29 @@
+The Google Cloud provider has introduced new write-only arguments for a more secure way to manage data. The new `WriteOnly` attribute accepts values from configuration and will not be stored in plan or state providing an additional layer of security and control over data access.
 
-For more information, see the [official HashiCorp documentation for Write-only Attributes](https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/write-only-arguments).
+For more information, see the [official HashiCorp documentation for Write-only Arguments](https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/write-only-arguments).
 
-The Google Cloud provider has added the following write-only attributes:
+The Google Cloud provider has added the following write-only arguments:
 - [`google_sql_user: password_wo`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_user#password-1)
 - [`google_secret_manager_secret_version: secret_data_wo`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/secret_manager_secret_version#secret_data_wo)
 - [`google_bigquery_data_transfer_config: sensitive_params.secret_access_key_wo`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_data_transfer_config#secret_access_key_wo)
 
-These were chosen due to them being marked as sensitive already in the provider. Although sensitive attributes do not appear in `terraform plan`, they are still stored in the Terraform state. Write-only attributes allow users to access and use data in their configurations without that data being stored in Terraform state.
+These were chosen due to them being marked as sensitive already in the provider. Although sensitive arguments do not appear in `terraform plan`, they are still stored in the Terraform state. Write-only arguments allow users to access and use data in their configurations without that data being stored in Terraform state.
 
-## Use the Google Cloud provider's new write-only attributes
+## Use the Google Cloud provider's new write-only arguments
 
-The following sections show how to use the new write-only attributes in the Google Cloud provider.
+The following sections show how to use the new write-only argument in the Google Cloud provider.
 
-### Applying a write-only attribute
+### Applying a write-only argument
 
-The following example shows how to apply a write-only attribute. All write-only attributes are marked with the `wo` suffix and can not be used with the attribute that it's mirroring. For example, `secret_data_wo` can not be used with `secret_data`.
+The following example shows how to apply a write-only argument. All write-only arguments are marked with the `wo` suffix and can not be used with the argument that it's mirroring. For example, `secret_data_wo` can not be used with `secret_data`.
```

Comment:
```suggestion
The following example shows how to apply a write-only argument. All write-only arguments are marked with the `wo` suffix and can't be used with the argument that it's mirroring. For example, `secret_data_wo` can not be used with `secret_data`.
```

nit: prefer can't over can not

https://developers.google.com/style/contractions

(I know that this documentation doesn't strictly follow the style guide but I find the advice to be useful personally & that the reasoning generally makes sense.)